long-read-rna-pipeline.wdl

```diff
@@ -2,13 +2,13 @@ version 1.0
 
 # ENCODE long read rna pipeline
 
-#CAPER docker quay.io/encode-dcc/long-read-rna-pipeline:v1.3
-#CAPER singularity docker://quay.io/encode-dcc/long-read-rna-pipeline:v1.3
-#CROO out_def https://storage.googleapis.com/encode-pipeline-output-definition/longreadrna.output_definition.json
```

Comment:
Similar to `get-splice-junctions.wdl`, there should probably be documentation that you need the latest caper for image autodetection.

Reply:
Thanks. Good points. Also noticed that the documentation is obsolete in few other points (such as statement that caper requires python 3.4 or newer.. I think nowadays it is 3.6).